Bootstrap/BootstrapTests/BootstrapTests.swift

```diff
@@ -39,13 +39,19 @@ class BootstrapTests: QuickSpec {
             }
 
             it("has a valid snapshot with model and OS in name and identifier") {
+                //expect(view).to(recordDeviceAgnosticSnapshot(identifier: "bootstrap"))
```

Comment:
Were these left in on purpose?

Reply:
Yes, to make easier to all devs record when the test fails